server/fleet/activities.go

```diff
@@ -1673,11 +1673,16 @@ func (a ActivityDeletedAppStoreApp) Documentation() (activity string, details st
 }
 
 type ActivityInstalledAppStoreApp struct {
-	HostID          int    `json:"host_id"`
+	HostID          uint   `json:"host_id"`
 	HostDisplayName string `json:"host_display_name"`
 	SoftwareTitle   string `json:"software_title"`
-	AppStoreID      int    `json:"app_store_id"`
+	AppStoreID      string `json:"app_store_id"`
 	CommandUUID     string `json:"command_uuid"`
+	Status          string `json:"status,omitempty"`
+}
+
+func (a ActivityInstalledAppStoreApp) HostIDs() []uint {
+	return []uint{a.HostID}
```

Comment:
Is it possible for this to ever be more than one element? I don't recall seeing this for other activities. Is it a required method for the activity interface or something?

Reply:
Yeah, it's required for automatically inserting into the host -> activity mapping table. See https://github.com/fleetdm/fleet/blob/main/server/datastore/mysql/activities.go#L62

Reply:
Ah, I forgot about that, thanks!